Sources/TuistGenerator/Generator/Generator.swift

```diff
@@ -18,7 +18,9 @@ public protocol Generating {
     /// - Parameters:
     ///     - project: The project to be generated.
     ///     - graph: The dependencies graph.
-    func generateProject(_ project: Project, graph: Graphing) throws -> AbsolutePath
+    ///     - sourceRootPath: The path all the files in the Xcode project will be realtived to. When it's nil, it's assumed that all the paths are relative to the directory that contains the manifest.
+    ///     - xcodeprojPath: Path where the .xcodeproj directory will be generated. When the attribute is nil, the project is generated at
```

Comment:
The comment seems to be not finished - generated where?:)

Reply:
Good catch. I just finished it.